src/app/forms/validators/custom-validators.ts

```diff
@@ -114,6 +114,10 @@ export class CustomValidators {
     };
   };
 
+  static alphanumeric(): ValidatorFn {
+    return this.customPattern(['^[a-zA-Z0-9]+$', 'must be alphanumeric']);
```

Comment:
Should use `*` instead of `+` in the regex so as to not make it required by extension.